src/pagination.rs

```diff
@@ -242,6 +247,142 @@ fn sort_posts(posts: &mut Vec<&liquid::value::Value>, config: &PaginationConfig)
     })
 }
 
+// Categories
+
+#[derive(Debug)]
+struct Category<'a> {
+    cat_name: liquid::value::Array,
```

Comment:
I'm assuming this is the "path" to this category?  I feel like a different name for the variable would be appropriate because this one, implies to me, the leaf categories' name (being singular).